packages/functional-tests/scripts/start-services.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -ex
```

Comment:
This was separated from `test-ci.sh`. Doing so gives us better timing data, so we can see startup time vs test time.

We should always be mindful of the cost of start vs the cost of actually running tests. If startup costs grows too high relative to test time, parallelizing starts to have diminishing returns.